src/components/WalletDropdown/MiniPortfolio/Activity/parseLocal.ts

```diff
@@ -182,24 +181,22 @@ export function parseLocalActivity(
   return { ...defaultFields, ...additionalFields }
 }
 
-export function useLocalActivities(): ActivityMap | undefined {
+export function useLocalActivities(account: string): ActivityMap {
   const allTransactions = useMultichainTransactions()
-  const { chainId } = useWeb3React()
   const tokens = useCombinedActiveList()
 
   return useMemo(
     () =>
-      chainId
```

Comment:
this check for useWeb3React.chainId was completely redundant / leftover from some refactor I made, so I removed it here